packages/core/src/Stores/traders-hub-store.js

```diff
@@ -388,6 +396,13 @@ export default class TradersHubStore extends BaseStore {
             return;
         }
 
+        if (this.financial_restricted_countries) {
+            this.available_mt5_accounts = this.available_cfd_accounts.filter(
+                account => account.market_type !== 'financial' && account.platform === CFD_PLATFORMS.MT5
+            );
+            return;
+        }
+
```

Comment:
@thisyahlen-deriv Do we need this filtering from FE side?  Should'nt the trading_platform_available _accounts API response do this?

Reply:
True @amina-deriv. But we are filtering the platforms from the FE side.

Reply:
so is BE response is still giving finanacial accounts for norway japan clients?

Reply:
Nope. Its giving gaming svg